shadow-cljs.edn

```diff
@@ -82,6 +82,7 @@
            :output-dir       "target/test"
            :optimizations    :simple
            :target           :node-test
+           :ns-regexp        "status-im.chat.models-test$"
```

Comment:
This is probably a left over from your local experiments.

Reply:
May I suggest you comment this out and add I quick explanation? This is quite handy for people running tests without the REPL when using `make test-watch`, so I think we could leave it here for other devs.